src/Utility_TEST.cc

```diff
@@ -488,6 +541,19 @@ TEST(MsgsTest, SetSphericalCoordinates)
   EXPECT_DOUBLE_EQ(2.2, msg.longitude_deg());
   EXPECT_DOUBLE_EQ(3.3, msg.elevation());
   EXPECT_DOUBLE_EQ(0.4, msg.heading_deg());
+
+  // For Moon's surface.
```

Comment:
Can you add a test for `msgs::Set` with a custom surface?

Reply:
Done https://github.com/gazebosim/gz-msgs/pull/279/commits/598d30de11a42f3775590fa30722cfa2dd104041